### ISSUE TITLE
Defaulting to the slightly slower but more stable QR method?

### ISSUE THREAD

**Issue by andreasnoack** [JuliaStats/GLM.jl#573]
Created from #500 to have a place to discuss if we want to make this change. This issue is closely related to #426 and #375.

I think I'm in favor of switching to using QR as the default. Thoughts?

cc @nalimilan @palday 

Blocked by https://github.com/JuliaStats/GLM.jl/issues/595

**Comment by nalimilan** [comment]
I agree. Better use the method that works in cases that do in other languages by default. That's one of main changes to make in 2.0 IMO.
